server/migrations/20190604161719-create-rating.js

```diff
@@ -0,0 +1,43 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Ratings', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
+    },
+    userId: {
+      type: Sequelize.INTEGER,
+      allowNull: false,
+      references: {
+        model: 'Users',
+        key: 'id'
+      }
+    },
+    articleId: {
+      type: Sequelize.INTEGER,
+      allowNull: false,
+      primaryKey: true,
+      references: {
+        model: 'Articles',
+        key: 'id',
+      },
+      onDelete: 'CASCADE',
+    },
+    ratings: {
```

Comment:
why do we allow null for this?

Also, how many times can a user rate an article?

if a user should be allowed to remove their rating from an article, consider using `paranoid:true` in the model definition (and having a `deletedAt` field here